src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -38,6 +38,8 @@ const (
 
 	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip=on"
 
+	splitIrqChipMachineOptions = "accel=kvm,kernel_irqchip=split"
+
```

Comment:
This change doesn't seem to belong here. Please move it to another patch.

Reply:
It's a simple cleanup. I've split it into a separate commit.